### DISCUSSION TITLE
[Feature Request] Midi Support

### DISCUSSION THREAD

**Discussion by RampedIndent** [theRealCarneiro/pulsemeeter#225] [Ideas]
Is there any plan to add midi support for changing volume levels, like that exists in voicemeeter?

**Comment by theRealCarneiro** [comment]
I've been working on a midi client for a while, only thing left to do is to create a GUI for it

**Comment by s1011205** [comment]
Whooaa, a midi client? Would that client let me press a key command on my MIDI keyboard and it could be picked up in Linux as a key command to be used for a hotkey in OBS, as I could do with MidiKey2Key in Windows? The lack of MidiKey2Key in Linux is one of the main hurdles to my doing OBS recording in Linux rather than Windows, given that I use my MIDI keyboard to switch OBS scenes.

**Comment by theRealCarneiro** [comment]
@s1011205  I'm making it so that you can also run regular linux commands, so you could do something like `xdotool key ctrl+i` and it would send a ctrl + i signal when you press the midi button

**Comment by s1011205** [comment]
Isso seria ótimo, Gabriel! Seriam mais dois obstaculos removidos ao alvo de poder gravar vídeo em Linux com OBS do mesmo jeito como faço em Windows.

**Comment by s1011205** [comment]
Thank you!

**Comment by leafayon** [comment]
Any news on this ?

**Comment by theRealCarneiro** [comment]
@leafayon ill take a look at this in the next couple of days

**Comment by leafayon** [comment]
Thank you for creating Pulsemeeter !

**Comment by trashcatt** [comment]
I just discovered this little program today and was wondering if there is any update on this? I'd love to be able to use my macropad to control inputs/outputs.
